### ISSUE TITLE
User permissions not letting me toggle as operator

### ISSUE THREAD

**Issue by couryrr** [itzg/docker-minecraft-bedrock-server#384]
## README.md update needed?
A note it seems that the [permissions](https://github.com/itzg/docker-minecraft-bedrock-server?tab=readme-ov-file#permissions) need to be updated. The docs have a space between the env and the value which is parsed wrong. There appears to need to be either no space or an equal. I can update the README.md if that makes sense.

## My issue.
The issue I am having is that I give my user xuid ops permissions and I can verify that on the container in the permissions.json. When I am in the game it looks like I have access to games setting specifically showing cords but the toggle won't toggle. It just makes the noise and nothing happens. Is there something else that needs to be done?

```
sudo docker run -d --rm --name mc_server -e EULA=TRUE -e OPS="not-gonna-say-this" -e DIFFICULTY="normal" -p 19132:19132/udp -v mc-volume:/data itzg/minecraft-bedrock-server
```

## Note to maintainer
This is great. My kid and I get to play minecraft and I am getting to teach him docker. If there is anything I can do to lend a hand please let me know.

**Comment by itzg** [comment]
You're right, the syntax in the README is wrong. That would be great if you to PR that change. 

As for the main issue at hand, you're saying that the permissions.json looks correct with the XUID you provided in OPS? Beyond that it might be a broader issue in the bedrock software itself that is preventing the desired outcome. It seems like Mojang changes a few recent versions ago that broke whitelist handling, so permissions might have been affected too.

**Comment by couryrr** [comment]
Seems like the case. The UI enables but nothing changes.

**Comment by swestcott** [comment]
Similar/same here. I can change drop-down settings e.g. game mode and difficulty, but none of the toggle settings work. More oddly, typing in the command _does_ work e.g. `/gamerule keepinventory true`

**Comment by couryrr** [comment]
> Similar/same here. I can change drop-down settings e.g. game mode and difficulty, but none of the toggle settings work. More oddly, typing in the command _does_ work e.g. `/gamerule keepinventory true`

This works for me as well. Seems like the ui is just borked.

**Comment by GentleHoneyLover** [comment]
Has anyone found a solution to settings toggles not toggling?
